build.gradle.kts

```diff
@@ -2,7 +2,7 @@ import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
 
 plugins {
   `maven-publish`
-  kotlin("jvm") version "1.4.10"
+  kotlin("jvm") version "1.5.10"
```

Comment:
https://kotlinlang.org/docs/whatsnew1520.html in case you want to update to todays new release ;)